package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.1",
   "description": "A modern, beautiful, customizable date picker for React",
   "main": "lib/index.js",
+  "types": "index.d.ts",
```

Comment:
What is reason to add types to package.json, I tested worked fine without that.

Reply:
I don't have any idea why it works but as mentioned in [TypeScript documentation](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html) we should add it.